### PR TITLE
fix: log Secret name instead of all data

### PR DESCRIPTION
**Pull request by zaunist**
### Type of change:

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

fix https://github.com/apache/apisix-ingress-controller/issues/1045

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

**Comment by tao12345666333** [comment]
sorry for delay. please resolve conflicts. thanks!

**Comment by zaunist** [comment]
@tao12345666333 PTAL, thanks.

File: pkg/providers/k8s/secret.go [hunk 1]
```diff
@@ -400,8 +400,8 @@ func (c *secretController) onUpdate(prev, curr interface{}) {
 		return
 	}
 	log.Debugw("secret update event arrived",
-		zap.Any("new object", curr),
-		zap.Any("old object", prev),
+		zap.Any("new object", curr.(*corev1.Secret).ObjectMeta.Name),
+		zap.Any("old object", prev.(*corev1.Secret).ObjectMeta.Name),
```
Comment (lingsamuel) [inline pkg/providers/k8s/secret.go:404]: we don't need to log secret name twice since it is immutable

Comment (zaunist) [reply pkg/providers/k8s/secret.go]: removed

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Comment by tao12345666333** [comment]
ping @lingsamuel for review, thanks

**Review by AlinsRan — APPROVED** [review]

**Review by lingsamuel — APPROVED** [review]

**Pull request merged**